auto/isolation

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) Igor Sysoev
+# Copyright (C) NGINX, Inc.
+
+# Linux clone syscall.
+
+nsflags="USER NS PID NET UTS IPC CGROUP"
+
+nxt_feature="clone(2)"
+nxt_feature_name=NXT_HAVE_CLONE
+nxt_feature_run=no
+nxt_feature_incs=
+nxt_feature_libs=
+nxt_feature_test="#include <sys/wait.h>
+                  #include <sys/syscall.h>
+
+                  int main() {
+                      return __NR_clone | SIGCHLD;
+                  }"
+. auto/feature
+
+if [ $nxt_found = yes ]; then
+    NXT_HAVE_CLONE=YES
+    
+    # Test all isolation flags
+    for flag in $nsflags; do
+        nxt_feature="CLONE_NEW${flag}"
+        nxt_feature_name=NXT_HAVE_CLONE_NEW${flag}
+        nxt_feature_run=no
+        nxt_feature_incs=
+        nxt_feature_libs=
+        nxt_feature_test="#define _GNU_SOURCE
+                          #include <sys/wait.h>
+                          #include <sys/syscall.h>
+                          #include <sched.h>
+
+                          int main() {
+                              return CLONE_NEW$flag;
+                         }"
+        . auto/feature
+
+        eval "NXT_HAVE_CLONE_NEW$flag=$nxt_found"
+    done
+else
+    NXT_HAVE_CLONE=NO
+fi
+
+NXT_ISOLATION=NO
+
+if [ $NXT_HAVE_CLONE = "YES" ]; then 
+    # test if at least one isolation flag was detected
+    for flag in $nsflags; do
```

Comment:
Do we really need a second loop here instead of just checking `$nxt_found` inside the previous one?

Reply:
actually, no =)